helm/ffc-demo-api-gateway/templates/deployment.yaml

```diff
@@ -1,26 +1,26 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: {{ .Values.name }}
+  name: {{ quote .Values.name }}
 spec:
   replicas: {{ .Values.replicaCount }}
   minReadySeconds: {{ .Values.minReadySeconds }}
   template:
     metadata:
       labels:
-        app: {{ .Values.name }}
-        environment: {{ .Values.environment }}
+        app: {{ quote .Values.name }}
+        environment: {{ quote .Values.environment }}
       annotations:
-        redeployOnChange: "{{ .Values.container.redeployOnChange }}"
+        redeployOnChange: "{{ quote .Values.container.redeployOnChange }}"
```

Comment:
I think this might be the cause of the build error: we've still got quotes around a value that's quoted using the Helm `quote` function.